lesson_07/homework_07.py

```diff
@@ -1,65 +1,93 @@
 # task 1
 """ Задача - надрукувати табличку множення на задане число, але
 лише до максимального значення для добутку - 25.
-Код майже готовий, треба знайти помилки та випраавити\доповнити.
+Код майже готовий, треба знайти помилки та випраавити/доповнити.
 """
+
+
 def multiplication_table(number):
-    # Initialize the appropriate variable
     multiplier = 1
-
-    # Complete the while loop condition.
-    while multiplier <= number:
+    while multiplier:
         result = number * multiplier
-        # десь тут помила, а може не одна
-        if  result > "25":
-            # Enter the action to take if the result is greater than 25
-            pass
+        if result >= 25:
+            break
         print(str(number) + "x" + str(multiplier) + "=" + str(result))
 
-        # Increment the appropriate variable
-        multi += 1
+        multiplier += 1
 
-multiplication_table(3)
-# Should print:
-# 3x1=3
-# 3x2=6
-# 3x3=9
-# 3x4=12
-# 3x5=15
 
+multiplication_table(3)
 
 # task 2
 """  Написати функцію, яка обчислює суму двох чисел.
 """
 
 
+def get_sum_numbers(*, first_number: int, second_number: int) -> int:
+    return first_number + second_number
+
+
+print(get_sum_numbers(first_number=3, second_number=7))
+
 # task 3
 """  Написати функцію, яка розрахує середнє арифметичне списку чисел.
 """
 
+
+def calculate_average(numbers_list) -> int:
+    total = sum(numbers_list)
+    count = len(numbers_list)
+    average = total // count
```

Comment:
```suggestion
    average = total / count
```
Середне арифметичне - не обов'язково ціле